aspnetcore/fundamentals/host/generic-host.md

```diff
@@ -1,20 +1,22 @@
 ---
-title: .NET Generic Host
+title: .NET Generic Host in ASP.NET Core
 author: rick-anderson
-description: Learn about the .NET Core Generic Host, which is responsible for app startup and lifetime management.
+description: Use .NET Core Generic Host in ASP.NET Core apps.  Generic Host is responsible for app startup and lifetime management.
 monikerRange: '>= aspnetcore-2.1'
 ms.author: riande
 ms.custom: mvc
 ms.date: 4/17/2020
 no-loc: ["ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
 uid: fundamentals/host/generic-host
 ---
-# .NET Generic Host
+# .NET Generic Host in ASP.NET Core
 
 ::: moniker range=">= aspnetcore-5.0"
 
 The ASP.NET Core templates create a .NET Core Generic Host (<xref:Microsoft.Extensions.Hosting.HostBuilder>).
 
+This topic provides information on using .NET Generic Host in ASP.NET Core. For information on using .NET Generic Host in a .NET app or .NET console app, see [.NET Generic Host](/dotnet/core/extensions/generic-host).
```

Comment:
@IEvangelist  I'm just getting started but would like to:

* Know the language you prefer.
* Drive traffic to your docs

Please suggest better wording I can use in all the extensions docs you've created. ASP.NET Core apps are .NET apps, so I'm not happy with the wording. I could go with something like 

for non-ASP.NET Core .NET apps
but that's worse.

Reply:
I prefer .NET apps, as they are all just .NET apps. I believe that @tdykstra had put together a doc that discussed which language to use when - Tom, could you please share that with Rick?

Reply:
The naming document is about how to refer to .NET 5/Core in general, without regard to app type. My suggestion for this text is:

```suggestion
This topic provides information on using .NET Generic Host in ASP.NET Core. For information on using .NET Generic Host in console apps, see [.NET Generic Host](/dotnet/core/extensions/generic-host).
```

Also, I wouldn't limit this text to a .NET 5 moniker block, it should apply to 3.1 as well.

Reply:
> Also, I wouldn't limit this text to a .NET 5 moniker block, it should apply to 3.1 as well.

I'll add it to both once it's approved.